### PR TITLE
Security

### PR DESCRIPTION
**Pull request by vibhuti910**

### CODE REVIEW

File: src/mtool/api/rest/blueprints/auth.py [hunk 1]
```diff
@@ -11,12 +11,12 @@
 @auth_bp.route('/api/v1.0/login/', methods=['POST'])
 def login():
     body_unicode = request.data.decode('utf-8')
-    print(request.authorization)
+    #print(request.authorization)
```
Comment (aswinssir) [inline src/mtool/api/rest/blueprints/auth.py:14]: Remove the commented code

File: src/mtool/api/rest/blueprints/home.py [hunk 2]
```diff
@@ -7,11 +7,11 @@
 @home_bp.route('/<path:path>')
 def serve(path):
     if path != "" and os.path.exists("public/" + path):
-        print('path debug in if', path)
-        print(os.path.exists("public/" + path))
+        #print('path debug in if', path)
```
Comment (aswinssir) [inline src/mtool/api/rest/blueprints/home.py:10]: Remove commented code

**Review by aswinssir — COMMENTED** [review]
Please remove the code that is commented. Also, please provide a more elaborate commit message and Description

**Review by aswinssir — APPROVED** [review]
LGTM

**Review by manoj22-k — APPROVED** [review]
LGTM

**Pull request merged**